ndcube/__init__.py

```diff
@@ -8,4 +9,4 @@
 except ImportError:
     __version__ = "unknown"
 
-__all__ = ['NDCube', 'NDCubeSequence', "NDCollection", "ExtraCoords"]
+__all__ = ['NDCube', 'NDCubeSequence', "NDCollection", "ExtraCoords", "GlobalCoords"]
```

Comment:
I don't think we want this in this file. I don't think we will ever want the user to instantiate one of these themselves.
```suggestion
__all__ = ['NDCube', 'NDCubeSequence', "NDCollection", "ExtraCoords"]
```